style.css

```diff
@@ -0,0 +1,174 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@300&display=swap');
+body {
+    font-family: 'Poppins', sans-serif;
+    font-family: 'Raleway', sans-serif;
+    background-color: var( --bg-color);
+    color: white;
+}
+
+:root {
+    --bg-color: #1a202c;
+    --bg-color: #1f2937;
+}
+
+.ham-menu {
+    display: none;
+}
+
+.navbar {
+    background-color: var( --bg-color);
+    color: white;
```

Comment:
use color variable

Reply:
Thankyou for the suggestion. Made the changes